Add a `<p>` about where the JSON file should come from

kienstra opened the pull request:
This adds some more explanation to the Block Lab importer:

>This JSON file should come from the export link or bulk action in the Content Blocks screen, not from the main Export tool.



<img width="927" alt="this-is-added" src="https://user-images.githubusercontent.com/4063887/69755037-f37c7d80-111c-11ea-88c4-9f270fab5cb3.png">

This clarifies that it's not possible to export the blocks from here:

<img width="587" alt="import-from-here" src="https://user-images.githubusercontent.com/4063887/69755098-19a21d80-111d-11ea-8492-cc35109526e8.png">

...and then import them using this:

<img width="968" alt="sing-impor" src="https://user-images.githubusercontent.com/4063887/69755195-4f470680-111d-11ea-836f-22618ec9d6c0.png">


Maybe this isn't the best wording, feel free to suggest something else.

kienstra committed:
Add a <p> about where the JSON file should come from

This clarifies that it can't come from
the main Export tool.

kienstra commented:
Hi @lukecarbis,
Could you please review this when you have a chance? It's not a very high priority, in my opinion.

lukecarbis reviewed (changes requested):
@kienstra This is great. My only suggestion is that we shouldn't link to the Export Tool. It doesn't make sense to provide a way of quickly accessing a discouraged action. Someone skimming might accidentally click the wrong link.

kienstra commented:
That's a good point, we shouldn't encourage going to the Export tool.

kienstra committed:
Remove the link for the Export tool

As Luke mentioned, a link is probably not good,
as we don't want people to go to that.

kienstra commented:
There's now no link to the 'Export tool':

<img width="955" alt="here-link" src="https://user-images.githubusercontent.com/4063887/69924010-7f204200-146e-11ea-93e1-979d0c6912df.png">
